tools/backups2datalad.py

```diff
@@ -0,0 +1,301 @@
+__requires__ = ["boto3", "click", "dandi", "datalad", "requests"]
+
+"""
+IMPORTANT NOTE ABOUT GIT CREDENTIALS
+
+This script uses `datalad create-sibling-github` to create GitHub repositories
+that are then pushed to GitHub.  The first step requires either GitHub user
+credentials stored in the system's credentials store or else a GitHub OAuth
+token stored in the global Git config under `hub.oauthtoken`.  In addition,
+pushing to the GitHub remotes happens over SSH, so an SSH key that has been
+registered with a GitHub account is needed for the second step.
```

Comment:
this https://github.com/datalad/datalad/wiki/FAQ#how-to-configure-insecure-local-credentials-store could be of help in cases of remote/server setups